rotkehlchen/exchanges/coinbase.py

```diff
@@ -345,7 +456,7 @@ def _api_query(
                 raise RemoteError(f'Coinbase json response does not contain data: {response.text}')
 
             # `data` attr is a list in itself
-            all_items.extend(json_ret['data'])
+            all_items.extend(json_ret['data'])  # `data` attr is a list in itself
```

Comment:
If you move `# `data` attr is a list in itself` here then delete it from the line above, please.

Reply:
Done! That was an oversight from my end.